STUDIO/org.openl.rules.test/test-resources/functionality/EPBDS-14557_Vocabulary_Validation.xlsx.msg.txt

```diff
@@ -0,0 +1,13 @@
+ERROR: Object 'F' is outside of valid domain '[Male, Female]'. The comparison always returns false.
+ERROR: Object '5' is outside of valid domain '[100, 200, 300]'. The comparison always returns false.
+ERROR: Object 'F' is outside of valid domain '[Male, Female]'. The comparison always returns false.
+ERROR: Object '5' is outside of valid domain '[100, 200, 300]'. The comparison always returns false.
+ERROR: Method 'GenderFactor(java.lang.String)' is not found.
```

Comment:
I am expecting, the test should find all methods. So verify that tests has correct inputs count.